src/_includes/layouts/base.html

```diff
@@ -65,12 +65,10 @@
     <link rel="preconnect" href="https://www.googletagmanager.com/">
 
     <script>
-        (function() {
```

Comment:
Seems like we should keep this to avoid creating a global? (Same variable name as in theme.js)

Reply:
Agreed

Reply:
Removed thanks :)